app/views/candidate_interface/satisfaction_survey/complexity.html.erb

```diff
@@ -1 +1,18 @@
-I found this service unnecessarily complex
+<% content_for :title, t('page_titles.complexity') %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_satisfaction_survey_recommendation_path) %>
+
+<%= form_with model: @survey, url: candidate_interface_satisfaction_survey_submit_complexity_path, method: :post do |f| %>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h2 class="govuk-heading-xl">
+        <%= t('page_titles.complexity') %>
+      </h2>
+
+      <%= f.hidden_field :question, value: t('page_titles.complexity') %>
```

Comment:
Instead of passing this using a hidden field, we could infer it from the route; when we're on the `post /complexity` route, we know which question they're trying to answer.

Reply:
That's a really good shout 👍